docs/source/introduction.nblink

```diff
@@ -1,3 +1,8 @@
 {
-  "path": "../../notebooks/introduction.ipynb"
+  "path": "../../notebooks/introduction.ipynb",
+  "extra-media": [
+    "../../notebooks/images",
+    "../../notebooks/python_logo.png",
```

Comment:
Note that this image is licensed under CC-BY-SA, which means we need to include the license and attribution (both in the source, and in the final docs). I would very much prefer to use a public domain (e.g. CC0) image, or one we create our selves.

Reply:
I replaced it with a smiley I did draw myself.
Thanks again for the pointer, I'm not too firm with the legal things and thought it is fine as long as you give props to the creator.